pom.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.github.openjverein</groupId>
+  <artifactId>jverein</artifactId>
+  <version>3.0.3</version>
+
+  <properties>
+    <project.build.sourceEncoding>ISO-8859-1</project.build.sourceEncoding>
+    <maven.compiler.release>11</maven.compiler.release>
+    <maven.test.skip>true</maven.test.skip>
+    <jameica.version>V_2_10_5_BUILD_488</jameica.version>
+    <hibiscus.version>V_2_10_24_BUILD_388</hibiscus.version>
+    <!-- See https://github.com/willuhn/jameica/commit/1a8a8005e04c6201972a63bd403b161f9e3802e7 -->
+    <swt.version>4.33</swt.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.activation</groupId>
+      <artifactId>activation</artifactId>
+      <version>1.1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.beanshell</groupId>
+      <artifactId>bsh-core</artifactId>
+      <version>2.0b4</version>
```

Comment:
Hier hatten wir schon die neue BSH 3, zumindest in feature-3.1.0, siehe #615.

Reply:
Die war interessanterweise im master nicht unter diesem Namen enthalten.

Reply:
Ja, weil wir das noch im nightly testen und nicht mit den bugfixes schon als 3.0.x verteilen wollen. Dieser PR basiert ja noch auf master. Daher hast du das nicht sehen können.

Reply:
Ich würde es hier auch nicht anpassen. Sobald der pr übernommen ist, kann angepasst werden. Außerdem wird uns renovate und dependabot einiges erzählen, vermute ich.